src/sentry/integrations/jira/notify_action.py

```diff
@@ -160,15 +161,16 @@ def after(self, event, state):
         def create_issue(event, futures):
             """Create the Jira ticket for a given event"""
 
-            # TODO check if a Jira ticket already exists for the given event's issue. if it does, skip creating it
-            resp = installation.create_issue(self.data)
-            ExternalIssue.objects.create(
-                organization_id=organization.id,
-                integration_id=integration.id,
-                key=resp["key"],
-                title=event.title,
-                description=installation.get_group_description(event.group, event),
-            )
+            # HACK to get fixVersion in the correct format
```

Comment:
@mgaeta and I went down a rabbithole to figure out that `fixVersions` was getting mangled, but this should be dealt with in a future PR to remove the hack